cs/general.md

```diff
@@ -29,7 +29,7 @@ In some projects (like Desktop, Android, iOS, Focus), the language will be direc
 
 ## Natural expression
 
-Any translation has to obey rules of the Czech grammar. Sentences have to be both [grammatically correct](#grammar) an also have a clear meaning. Avoid any archaisms or long and complicated sentences and constructions. Especially beware of translating the English original word by word. Such translations are inaccurate and sound "like a robot". Instead use your own word to express the desired meaning. Be careful about commas and semicolons.
+Any translation has to obey rules of the Czech grammar. Sentences have to be both [grammatically correct](#grammar) an also have a clear meaning. Avoid any archaisms or long and complicated sentences and constructions. Especially beware of translating the English original word by word. Do not stick to the English word order nor sentence constructions too much. Such translations are inaccurate and sound "like a robot". Instead use your own words to express the desired meaning. For complex sentences, read them multiple times before translating to Czech. To make the translated text sound natural, it may be necessary to adjust the number of sentences (either split them or combine) or change their order and relations. Be careful about commas and semicolons too.
```

Comment:
Missing "d":
"an also have a clear meaning" =>  "and also have a clear meaning"

Not sure about the word order here, I would change it slightly:
"Instead use your own words to express the desired meaning" => "Use your own words to express the desired meaning instead"

Reply:
Thanks @FHRNet. I have made the updates.